app/auth/middleware.test.js

```diff
@@ -56,7 +51,7 @@ describe('Authentication middleware', function() {
         expect(nextSpy).to.be.calledOnceWithExactly()
       })
 
-      it('doesn’t regenerate the session', function() {
+      it('doesnt regenerate the session', function() {
```

Comment:
Any reason to remove the apostrophe here?

Reply:
being trigger happy :)